### ISSUE TITLE
fixed f1 score后的TextCNN_train.py中出现操作ValueError

### ISSUE THREAD

**Issue by GUOXINTONG** [brightmart/text_classification#119]
line 137: predict = []
line 145: predict += logits[0]

会报错：operands could not be broadcast together with shapes (0,) (n,)
(n是label个数)

原因应该是在循环中空数组和预测值logits的第0维相加出现了维度问题。
我改成了　line 137: predict = np.zeros(n)　解决
不知道只是我的bug吗还是？

**Comment by lilyhong420** [comment]
我也出现了这样的问题

**Comment by TingNLP** [comment]
不太明白为何只加logits[0]
应该把logits全部的结果列表起来与vaild_y计算F1才是？

另外logits的值是sigmoid后的结果？
